pkg/collector/corechecks/snmp/fetch/fetch_column.go

```diff
@@ -59,34 +64,34 @@ func fetchColumnOids(session sessionAPI, oids map[string]string, bulkMaxRepetiti
 			columnOids = append(columnOids, k)
 			requestOids = append(requestOids, v)
 		}
-		// sorting columnOids and requestOids to make them deterministic for testing purpose
+		// sorting ColumnOids and requestOids to make them deterministic for testing purpose
 		sort.Strings(columnOids)
 		sort.Strings(requestOids)
 
-		results, err := getResults(session, requestOids, bulkMaxRepetitions)
+		results, err := getResults(sess, requestOids, bulkMaxRepetitions)
 		if err != nil {
 			return nil, err
 		}
-		newValues, nextOids := resultToColumnValues(columnOids, results)
+		newValues, nextOids := gosnmplib.ResultToColumnValues(columnOids, results)
 		updateColumnResultValues(returnValues, newValues)
 		curOids = nextOids
 	}
 	return returnValues, nil
 }
 
-func getResults(session sessionAPI, requestOids []string, bulkMaxRepetitions uint32) (*gosnmp.SnmpPacket, error) {
+func getResults(sess session.Session, requestOids []string, bulkMaxRepetitions uint32) (*gosnmp.SnmpPacket, error) {
 	var results *gosnmp.SnmpPacket
-	if session.GetVersion() == gosnmp.Version1 {
+	if sess.GetVersion() == gosnmp.Version1 {
```

Comment:
strange renaming...

Reply:
That's to avoid name collision with the new `session` package.